pkg/operator/ceph/cluster/mgr/mgr.go

```diff
@@ -120,6 +124,15 @@ func (c *Cluster) Start() error {
 		return errors.Wrap(err, "failed to check for mgrs to skip reconcile")
 	}
 
+	shouldRotateKeys, err := shouldRotateMgrKeys(c.context, c.clusterInfo)
+	if err != nil {
+		return errors.Wrapf(err, "failed to check if cephx keys for mgr daemons in the namespace %q should be rotated", c.clusterInfo.Namespace)
+	}
+	c.shouldRotateCephxKeys = shouldRotateKeys
+	if c.shouldRotateCephxKeys {
+		logger.Infof("cephx keys for mgr daemons in the namespace %q will be rotated", c.clusterInfo.Namespace)
```

Comment:
```suggestion
		logger.Debugf("cephx keys for mgr daemons in the namespace %q will be rotated", c.clusterInfo.Namespace)
```
nit

Reply:
Why `debug`?  This log will only be printed if key rotation is required. It won't be printed on every reconcile. So I strongly feel it should be a `info` and not a `debug`.